_episodes/00-introduction.md

```diff
@@ -8,7 +8,7 @@ questions:
 objectives:
 - "Describe the format of a study group session."
 - "Find notes from previous sessions."
-- "Summarise the responsibilities of the Discussion Lead and Notetaker for each session."
+- "Summarize the responsibilities of the Discussion Lead and Notetaker for each session."
```

Comment:
In accordance with [The Carpentries Style Guide](https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html#), I am using British English spelling, so will change these back to "-ise" endings.